ArmPlatformPkg/PeilessSec/PeilessSec.inf

```diff
@@ -43,6 +43,7 @@
   DebugAgentLib
   DebugLib
   HobLib
+  LzmaDecompressLib

```

Comment:
This change was deliberate. LzmaDecompressLib is not a library class, and the module does not strictly depend on it.

In the integration, it may be the case that LzmaDecompressLib is required, when the payload is compressed using LZMA. So LzmaDecompressLib may be incorporated into the module using NULL library class resolution.

E.g.,
```
Platform/ARM/JunoPkg/ArmJuno.dsc-  ArmPlatformPkg/PeilessSec/PeilessSec.inf {
Platform/ARM/JunoPkg/ArmJuno.dsc-    <LibraryClasses>
Platform/ARM/JunoPkg/ArmJuno.dsc:      NULL|MdeModulePkg/Library/LzmaCustomDecompressLib/LzmaCustomDecompressLib.inf
Platform/ARM/JunoPkg/ArmJuno.dsc-  }
```

Reply:
Sure, I was going to go that approach, but I didn't see mention of that in the PR (I may well have just missed it), so I put this up to restore it.

I agree however, and I was surprised that it relied directly on LZMA. I'll drop this PR and update the platform, thanks.